contrib/inventory/ovirt.py

```diff
@@ -172,9 +172,9 @@ def get_ovirt_driver():
 
         # If the appropriate environment variables are set, they override
         # other configuration; process those into our args and kwargs.
-        kwargs['url'] = os.environ.get('OVIRT_URL')
-        kwargs['username'] = os.environ.get('OVIRT_EMAIL')
-        kwargs['password'] = os.environ.get('OVIRT_PASS')
+	kwargs['url'] = environ.get('OVIRT_URL') if 'OVIRT_URL' in os.environ.keys() else kwargs['url']
```

Comment:
There a missing space here.

Are you sure `environ` is defined here ? I don't see it.

Just FTR - and I'm not opinionated on this - other possible alternative syntaxes are:

    kwargs['url'] = environ.get('OVIRT_URL', kwargs['url'])
    kwargs['url'] = environ.get('OVIRT_URL') or kwargs['url']